sites/industrial-lasers/server/templates/index.marko

```diff
@@ -35,15 +35,15 @@ $ const adSlots = {
       <div class="row">
         <div class="col-lg-6 mb-block">
           <endeavor-content-query-section-list
-            limit=4
+            limit=2
```

Comment:
Would it be more consistent if the limit was set to 3 - to match the row below?

Reply:
I set it to 2 because it left a smaller whitespace gap, but I can change it to 3 if you don't mind a larger gap